### PR TITLE
feature: update the roadmap for a new stage

### PR DESCRIPTION
**Pull request by Starnop**
Signed-off-by: Starnop <starnop@163.com>

### Ⅰ. Describe what this PR did
The `Dragonfly` was written with a unified programming language now and it has entered a new stage. So we need to redefine the roadmap to do some new plan for Dragonfly.

### Ⅱ. Does this pull request fix one issue?

None.

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
None.

### CODE REVIEW

**Comment by Starnop** [comment]
@allencloud @lowzj PTAL. THX>

**Comment by lowzj** [comment]
LGTM

**Comment by allencloud** [comment]
LGTM

File: ROADMAP.md [hunk 1]
```diff
@@ -1,103 +1,104 @@
...
 
-* Support multi-registrys.
-* Support supernode cluster. Cluster the SuperNode to decrease possibility of failure.
-* Supernode support storage driver.
-* Use [IPFS](https://github.com/ipfs/go-ipfs) to share block datas between SuperNodes.
-* Highly user-customized modules.
+## v1.0.x
 
-## Future
+This is a stable version in GA stage. And we will focus on making `Dragonfly` support more scenarios. Including:
```
Comment (zhouhaibing089) [inline ROADMAP.md]: I assume when we say GA, it means there is no user impact going forward: no behavior change, features are not going to be deprecated, etc. 

I feel somehow, we should track all the features stage somehow, not all of them can go GA at the same time. For example, the core it pretty stable, but the plugin framework might still be in alpha.

**Pull request merged**